pixel-avatar-hardhat/package.json

```diff
@@ -4,28 +4,30 @@
   "description": "",
   "main": "index.js",
   "scripts": {
+    "chain": "hardhat node",
+    "deploy": "hardhat run scripts/deploy.ts --network localhost",
+    "deploy:local": "hardhat run scripts/deploy.ts --network localhost",
+    "deploy:rinkeby": "hardhat run scripts/deploy.ts --network rinkeby",
+    "compile": "hardhat compile",
     "test": "hardhat test"
   },
   "keywords": [],
   "author": "Developer DAO",
+  "private": true,
```

Comment:
Any reason why this line was added?

Reply:
Ow, I don't remember @Dhaiwat10 
I can remove that